query_builder.go

```diff
@@ -0,0 +1,69 @@
+package pgq
```

Comment:
Pls, move it to `internal`, so we don't export these symbols

https://github.com/dataddo/pgq/tree/main/internal